R/board_connect.R

```diff
@@ -46,51 +46,51 @@
 #'  using `"manual"` if `server` and `key` are provided, `"envvar"` if both
 #'  environment variables are set, and `"rsconnect"` otherwise.
 #' @param server For `auth = "manual"` or `auth = 'envvar'`, the full url to the server,
-#'   like `http://server.rstudio.com/rsc` or `https://connect.rstudio.com/`.
-#'   For `auth = 'rsconnect'` a host name used to disambiguate RSC accounts,
-#'   like `server.rstudio.com` or `connect.rstudio.com`.
-#' @param account A user name used to disambiguate multiple RSC accounts.
-#' @param key The RStudio Connect API key.
+#'   like `http://server.posit.co/rsc` or `https://connect.posit.co/`.
+#'   For `auth = 'rsconnect'` a host name used to disambiguate Connect accounts,
+#'   like `server.posit.co` or `connect.posit.co`.
+#' @param account A user name used to disambiguate multiple Connect accounts.
+#' @param key The Posit Connect API key.
 #' @param output_files `r lifecycle::badge("deprecated")` No longer supported.
 #' @family boards
 #' @export
 #' @examples
 #' \dontrun{
-#' board <- board_rsconnect()
+#' board <- board_connect()
 #' # Share the mtcars with your team
 #' board %>% pin_write(mtcars, "mtcars")
 #'
 #' # Download a shared dataset
 #' board %>% pin_read("timothy/mtcars")
 #' }
-board_rsconnect <- function(auth = c("auto", "manual", "envvar", "rsconnect"),
-                            server = NULL,
-                            account = NULL,
-                            key = NULL,
-                            output_files = FALSE,
-                            cache = NULL,
-                            name = "rsconnect",
-                            versioned = TRUE,
-                            use_cache_on_failure = is_interactive(),
-                            versions = deprecated()) {
+board_connect <- function(auth = c("auto", "manual", "envvar", "rsconnect"),
+                          server = NULL,
+                          account = NULL,
+                          key = NULL,
+                          output_files = FALSE,
+                          cache = NULL,
+                          name = "posit-connect",
+                          versioned = TRUE,
+                          use_cache_on_failure = is_interactive(),
+                          versions = deprecated()) {
 
   server <- rsc_server(auth, server, account, key)
 
   if (lifecycle::is_present(versions)) {
-    lifecycle::deprecate_warn("1.0.0", "board_rsconnect(versions)","board_rsconnect(versioned)")
+    lifecycle::deprecate_warn("1.0.0", "board_connect(versions)","board_connect(versioned)")
     versioned <- versions
   }
 
-  cache <- cache %||% board_cache_path(paste0("rsc-", hash(url)))
+  cache <- cache %||% board_cache_path(paste0("connect-", hash(url)))
```

Comment:
I think this change will be fine. Any better idea for a name for the cache on disk?

Reply:
Nope 😞